src/cobalt_model/pagination_config.rs

```diff
@@ -0,0 +1,108 @@
+use liquid;
+use std::borrow::Cow;
+use std::vec::Vec;
+use super::SortOrder;
+
+// struct Trails {
+//   before: i32,
+//   after: i32,
+// }
+
+#[derive(Clone, Debug)]
+pub(crate) struct PaginationCfg {
+  pub(crate) include: String,
+  pub(crate) per_page: i32,
+  pub(crate) permalink: String,
```

Comment:
Isn't this duplicating the `permalink` in the frontmatter?

Reply:
It is indeed a struct representation of the `pagination` field from the frontmatter.

Reply:
Shall I remove this struct?

Reply:
I'm a little confused by this comment.  Is `frontmatter.pagination.permalink` duplicate of `frontmatter.permalink`?  If not, what unique role do they fulfill?

Reply:
Oh I see! `frontmatter.pagination.permalink` is the one we defined for pagination only, to configure where the different indexes will go, not related to `frontmatter.permalink`.